metal/end_model/end_model.py

```diff
@@ -111,7 +111,8 @@ def _make_layer(self, module, output_dim=None):
         if isinstance(module, IdentityModule):
             return module
         layer = [module]
-        layer.append(nn.ReLU())
+        if self.config["relu"]:
+            layer.append(nn.ReLU())
```

Comment:
I think the cleaner fix is to deal with the no-middle layers case properly, as we discussed; I'll work on that